.github/workflows/notion-sync.yaml

```
@@ -0,0 +1,30 @@
+name: Notion Sync
+
+permissions:
+  issues: read
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - opened
+      - edited
+      - labeled
+      - unlabeled
+      - assigned
+      - unassigned
+      - milestoned
+      - demilestoned
+      - reopened
+      - closed
+
+jobs:
+  notion_job:
+    runs-on: ubuntu-latest
+    name: Add GitHub Issues to Notion
+    steps:
+      - name: Add GitHub Issues to Notion
+        uses: tryfabric/notion-github-action@v1
```

We need to always be pinning specific action versions. 

```suggestion
        uses: tryfabric/notion-github-action@10c6b128faeb9a1f16efc65a0b388b7595047e62
```